### PR TITLE
Exercise1_Echo_jbuican

### PR DESCRIPTION
**Pull request by jbuican14**
Modify function echo and print out arguments to index.html file

### CODE REVIEW

**Comment by jbuican14** [comment]
I am not sure why it happens but I will try to do a pull request again.

**Comment by afeld** [comment]
> I am not sure why it happens

Not sure what you're referring to...?

> I will try to do a pull request again.

No need! You can continue to synchronize (push) as you have been, within this same pull request.

File: echo.js [hunk 1]
```diff
@@ -1,7 +1,25 @@
...
+  function echo() {
 
-echo();
-echo('bla');
-echo('foo', 'bar', 'baz');
+    var createElement = document.createElement('p');
+    var innerContent = [];
+    var count = 0;
+    //if arguments is not zero
+    if (arguments.length !== 0) {
```
Comment (afeld) [inline echo.js:9]: Is this check necessary? What would happen if it wasn't here?

File: echo.js [hunk 2]
```diff
@@ -1,7 +1,25 @@
...
+      for (var i in arguments) {
+        innerContent[count] = "'" + arguments[i] + "'";
+        createElement = document.createTextNode(innerContent[count]);
+        var linebreak = document.createElement("br");
+        document.body.appendChild(createElement);
+        document.body.appendChild(linebreak);
+        count++;
+      }
+    } else {
+      innerContent = '';
```
Comment (afeld) [inline echo.js:19]: Why do this assignment to `innerContent`? Is it being used after this line?

File: echo.js [hunk 3]
```diff
@@ -1,7 +1,25 @@
...
-}
+onload = function() {
+  // echo function is to print out each argument on a new line
+  function echo() {
 
-echo();
-echo('bla');
-echo('foo', 'bar', 'baz');
+    var createElement = document.createElement('p');
+    var innerContent = [];
```
Comment (afeld) [inline echo.js:6]: Why make this an array? Do you ever need to reference more than the current element's `innerContent`?

File: echo.js [hunk 4]
```diff
@@ -1,7 +1,25 @@
...
+    var count = 0;
+    //if arguments is not zero
+    if (arguments.length !== 0) {
+      for (var i in arguments) {
```
Comment (afeld) [inline echo.js:10]: Careful with using `for...in` with arrays (or `arguments`).

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in#Array_iteration_and_for...in

**Comment by afeld** [comment]
Getting an error when I load [the page](http://jbuican14.github.io/echo/)...why is that happening?

screen shot 2015-10-29 at 9 14 46 am

**Pull request closed without merging**